### PR TITLE
test(e2e): migrate portal tests from postman

### PR DESCRIPTION
**Pull request by gcusnieux**
gravitee-io/issues#7643

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/7643-portal-tests-run-e2e/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-e2e/api-test/portal/portal-authorizations.spec.ts [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    {
+      title: 'GET /pages/:pageId/content',
+      method: 'getPageContentByApiIdAndPageId',
+      api: apiPortalApi,
+      params: { apiId, pageId },
+    },
+    { title: 'POST /users/registration', method: 'registerNewUserRaw', api: usersApi, params: {} },
+    { title: 'POST /users/registration/_finalize', method: 'finalizeUserRegistrationRaw', api: usersApi, params: {} },
+    { title: 'POST /users/_rest_password', method: 'resetUserPasswordRaw', api: usersApi, params: {} },
+    { title: 'GET /categories', method: 'getPortalInformationRaw', api: portalApi },
```
Comment (marcambier) [inline gravitee-apim-e2e/api-test/portal/portal-authorizations.spec.ts]:
```suggestion
    { title: 'GET /info', method: 'getPortalInformationRaw', api: portalApi },
```

File: gravitee-apim-e2e/api-test/portal/portal-authorizations.spec.ts [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+    { title: 'GET /configuration/applications/identities', method: 'getPortalIdentityProvidersRaw', api: portalApi },
+    {
+      title: 'GET /configuration/applications/identities/:identityProviderId',
+      method: 'getPortalIdentityProviderRaw',
+      api: portalApi,
+      params: { identityProviderId: '' },
+    },
+    { title: 'GET /info', method: 'getPortalInformationRaw', api: portalApi },
+    { title: 'GET /pages', method: 'getPagesRaw', api: portalApi, params: {} },
+    { title: 'GET /pages/:pageId', method: 'getPagesByApiIdRaw', api: apiPortalApi, params: { apiId } },
```
Comment (marcambier) [inline gravitee-apim-e2e/api-test/portal/portal-authorizations.spec.ts]:
```suggestion
    { title: 'GET /pages/:apiId', method: 'getPagesByApiIdRaw', api: apiPortalApi, params: { apiId } },
```

File: gravitee-apim-e2e/api-test/portal/portal-authorizations.spec.ts (hunk 1 above)
Comment (marcambier) [reply gravitee-apim-e2e/api-test/portal/portal-authorizations.spec.ts]: But would be duplicated with line 90..

**Review by marcambier — CHANGES_REQUESTED** [review]

File: gravitee-apim-e2e/api-test/portal/portal-authorizations.spec.ts (hunk 2 above)
Comment (gcusnieux) [reply gravitee-apim-e2e/api-test/portal/portal-authorizations.spec.ts]: Oups. I removed both, it were duplicate

**Review by marcambier — APPROVED** [review]

**Pull request merged**